File: test/ifthen.pas
program ifthen;
　　var year:integer;
　　begin
　　　write("Input year:");read(year);
　　　write(year);
     write(6);
　　　if ((year mod 400)=0 )　then
　　　　write("is a leap year.")
　　　else
　　　　if ((year mod 4)=0)and(year mod 100<>0)
　　　　then write("is a leap year.")
　　　　else write("is not a leap year.");
　　end.

File: test/ifthen.c
#include <stdio.h>


int year;


int main() {
    printf("Input year:");
    scanf("%d", &year);
    printf("%d", year);
    printf("\n");
    if (((year % 400) == 0)) 
        printf("is a leap year.");
    else
        if (((year % 4) == 0) && (year % 100 != 0)) 
        {
            printf("is a leap year.");
        }
        else
            printf("is not a leap year.");
    return 0;
}